src/org/exist/xquery/GeneralComparison.java

```diff
@@ -1171,32 +1172,28 @@ protected void switchOperands()
         context.getProfiler().message( this, Profiler.OPTIMIZATIONS, "OPTIMIZATION", "Switching operands" );
 
         //Invert relation
-        switch( relation ) {
+        switch(relation) {
 
-            case Constants.GT: {
-                relation = Constants.LT;
+            case GT:
+                relation = Comparison.LT;
```

Comment:
will static import solve that compile errors? You have mix of patterns: short form, for example  `LT` and long one - `Comparison.LT`. Personally, I prefer short form.